Tests/js/views/services/assets/ez-locationviewviewservice-tests.js

```diff
@@ -430,6 +430,216 @@ YUI.add('ez-locationviewviewservice-tests', function (Y) {
+                Assert.areEqual(
+                    secondState, e.notification.state,
+                    "The notification state should be 'error'"
+                );
+                Assert.isString(
+                    e.notification.text,
+                    "The notification text should be a String"
+                );
+                Assert.areSame(
+                    timeout, e.notification.timeout,
+                    "The notification timeout should be set to 0"
+                );
+                Assert.areSame(
+                    'move-notification-' + parentLocationId + locationId,
+                    e.notification.identifier,
+                    "The notification identifier should match"
+                );
+            });
+        },
+
+        tearDown: function () {
+            this.service.destroy();
+            delete this.service;
+            delete this.app;
+            delete this.locationMock;
+            delete this.capiMock;
+            delete this.contentServiceMock;
+            delete this.responseMock;
+        },
+
+        "Should notify when trying to move a content but get an error": function () {
+            var that = this,
+                notified = false,
+                parentLocationMock = new Y.Mock(),
+                fakeEventFacade = {selection : {location : parentLocationMock }};
+
+            Y.Mock.expect(parentLocationMock, {
+                method: 'get',
+                args: ['id'],
+                returns: this.parentLocationId
+            });
+            this.error = true;
+            this.service.on('contentDiscover', function (e) {
+                e.config.contentDiscoveredHandler.call(this, fakeEventFacade);
+            });
+
+            this.service.once('notify', function (e) {
+                notified = true;
+                that._assertOnNotification(e,'started', 'error', 5, that.parentLocationId, that.locationId);
+            });
+            this.service.fire('whatever:moveAction');
+            Assert.isTrue(notified, "The notify event should have been fired");
+        },
+
+        "Should notify when trying to move a content without redirection": function () {
+            var that = this,
+                notified = false,
+                parentLocationMock = new Y.Mock(),
+                fakeEventFacade = {selection : {location : parentLocationMock }};
+
+
+            Y.Mock.expect(parentLocationMock, {
+                method: 'get',
+                args: ['id'],
+                returns: this.parentLocationId
+            });
+
+            this.service.on('contentDiscover', function (e) {
+                e.config.contentDiscoveredHandler.call(this, fakeEventFacade);
+            });
+
+            this.service.once('notify', function (e) {
+                notified = true;
+                that.activeView = new Y.View({location: 'newLocation/1/2/3/4'});
+                Y.Mock.expect(that.app, {
```

Comment:
it's a bit late to add expectations, usually expectations are added in the setUp or in the beginning of a test.